content/docs/reference/access-log-fields.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Access Log Fields
+description: Use Access Log Fields to display HTTP request logs from the proxy service.
+keywords: [access logs, proxy service, pomerium, http]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Access Log Fields
+
+## Summary
+
+The **Access Log Fields** setting displays HTTP request logs from the proxy service with a message of the type `http-request`.
+
+## How to Configure
+
+<Tabs>
+<TabItem label="Core" value="Core">
+
+| **Config file key** | **Environment variable** | **Type** | **Default** |
+| :-- | :-- | :-- | :-- |
+| `access_log_fields` | `ACCESS_LOG_FIELDS` | Array of strings | See [Access Fields and Defaults](#access-fields-and-defaults) |
+
+</TabItem>
+<TabItem label="Enterprise" value="Enterprise">
+
+Configure **Access Log Fields** in the Console:
+
+![Add access log fields in the Console](./img/http-logs/access-custom-logged-fields.gif)
+
+Add **Custom Headers** in the Console:
+
+![Add custom headers in the Console](./img/http-logs/access-custom-headers.gif)
+
+</TabItem>
+<TabItem label="Kubernetes" value="Kubernetes">
+
+| **Name** | **Type** | **Default** |
+| :-- | :-- | :-- |
+| `accessLogFields` | Array of strings | See [Access Fields and Defaults](#access-fields-and-defaults) |
+
+</TabItem>
+</Tabs>
+
+## Access Log Fields and Defaults
+
+The table below lists all available access log fields:
+
+| **Field name** | **Description** | **Default field** |
+| :-- | :-- | :-- |
+| `authority` | The HTTP Authority. Can be a hostname, domain name, or IP address and may contain a port number (for example, `www.example.com`) | Yes |
+| `duration` | The amount of time the request takes to complete in seconds | Yes |
+| `forwarded-for` | Identifies the originating IP address of the client | Yes |
+| `method` | The HTTP method, such as `GET`, `POST`, or `PUT` | Yes |
+| `path` | The HTTP path (for example, `/some/path`) | Yes |
+| `referer` | The HTTP Referer, or the address of the web page from which the resource has been requested | Yes |
+| `request-id` | The request's unique identifier | Yes |
+| `response-code` | The HTTP Response Code (for example `200` or `500`) | Yes |
+| `response-code-details` | The HTTP Response Code Details (for example, `File not found` or `404`) | Yes |
+| `size` | The HTTP Response size | Yes |
+| `upstream-cluster` | The cluster the request was sent to | Yes |
```

Comment:
We might want to mention here that the cluster name is assigned by Pomerium and/or give some examples of what this might look like (e.g. "route-3bb74f76f9d71f9c" or "pomerium-control-plane-http").